main.go

```diff
@@ -398,37 +403,49 @@ func usageMessage() {
 /* listAll = true - all versions including beta and rc will be displayed */
 /* listAll = false - only official stable release are displayed */
 func installOption(listAll bool, custBinPath, mirrorURL *string) {
-	tflist, _ := lib.GetTFList(*mirrorURL, listAll) //get list of versions
-	recentVersions, _ := lib.GetRecentVersions()    //get recent versions from RECENT file
-	tflist = append(recentVersions, tflist...)      //append recent versions to the top of the list
-	tflist = lib.RemoveDuplicateVersions(tflist)    //remove duplicate version
-
-	if len(tflist) == 0 {
-		fmt.Println("[ERROR] : List is empty")
-		os.Exit(1)
+	tflist, err := lib.GetTFReleases(*mirrorURL, listAll) //get list of versions
+	if err != nil {
+		log.Fatalf("Encountered error while getting list of releases\nError: %v", err)
+	}
+
+	recentVersions, err := lib.GetRecentVersions(*mirrorURL) //get recent versions from RECENT file
+	if err != nil {
+		log.Fatalf("Error while reading local versions file: %v", err)
+	}
+	tflist = append(recentVersions, tflist...)
+	tflist = lib.RemoveDuplicateVersions(tflist)
+
+	templates := promptui.SelectTemplates{
+		Active:   `> {{ .Version | green | bold }}`,
+		Inactive: `{{ .Version | cyan }}`,
+		Selected: `{{ "✔" | green | bold }} {{ .Version | cyan }}`,
```

Comment:
Just to keep compatibility with dump terminals can we make `✔` configurable via command line option like `--ascii` (as in when option is provided use ASCII char instead of `✔`? Or to not complicate code just replace it with some 7bit ASCII char? (I'm not sure which exact char would be a good fir though)

Reply:
I doubt many users will have a strong desire for customizing this but good point on making it a safer character.

Reply:
@yermulnik updated with https://github.com/warrensbox/terraform-switcher/commit/7346f8d07bbf111c65171769e9eec3b13839b8cb